docs/long-john/assembly_manual/assembly_manual.md

```diff
@@ -3,15 +3,15 @@
 ## Mechanical Assembly
 
 !!! info annotate "Threadlocker"
-    These machines vibrate an immense amount when in use and fasteners are bound to back out. Don’t be afraid to add a little bit of medium-strength threadlocker to your central screw to avoid this problem. Threadlocker should **not** be used on the self tapping screws as the chemicals in threadlocker may damage the ABS and cause it to weaken.
+    These machines vibrate intensely when running and fasteners are bound to back out. Don’t be afraid to add a little bit of medium-strength threadlocker to your central screw to avoid this problem. Threadlocker should **not** be used on the self-tapping screws as the chemicals in threadlocker may damage the ABS and cause it to weaken.
 
 ![Long-John Toolsetter assembly image](./img/assembly_image_1.png){: .shadow}
 
 ## Electronics Assembly
 
 !!! info annotate "Are you building an LDO kit?"
-    LDO have written a fantastic supplementary guide for the wiring included with their kit, which is pre-cut, pre-crimped, and pre-soldered! If you're building an LDO kit then check out their guide [here!](https://www.ldomotion.com/#/guide/Milo-CNC-V15-Wiring-Guide)
+    LDO has written a fantastic supplementary guide for the wiring included with their kit, which is pre-cut, pre-crimped, and pre-soldered! If you're building an LDO kit then check out their guide [here!](https://www.ldomotion.com/#/guide/Milo-CNC-V15-Wiring-Guide)
```

Comment:
"have" correct

Reply:
LDO is a singular entity. 
![4tgk06X 1](https://github.com/user-attachments/assets/555f7fe6-6b20-4e60-bdd3-4514cea16ab3)